sale_order_field_advanced/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     """,
     "author": "Solvos",
     "license": "LGPL-3",
-    "version": "15.0.1.0.0",
+    "version": "15.0.2.0.0",
```

Comment:
This is a minor improvement, not major, so better this:
```suggestion
    "version": "15.0.1.1.0",
```

I'll install it at customer Test environment anyway